keyboards/handwired/wwa/rosetta/info.json

```diff
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "spacehangover",
+    "keyboard_name": "Rosetta",
+    "maintainer": "spacehangover",
+    "development_board": "elite_c",
```

Comment:
Reading `readme.md` to confirm, does this board use Pro Micro development board, rather than Elite C development board?
If yes - please change `development_board` from `elite_c` to `promicro`